src/main/java/net/masterthought/cucumber/Configuration.java

```diff
@@ -81,6 +83,20 @@ public int getTrendsLimit() {
         return trendsLimit;
     }
 
+    /**
+     * Sets whether to treat pending and undefined steps as failed.
```

Comment:
I would like to have separate field for all three https://github.com/damianszczepanik/cucumber-reporting/blob/master/src/main/java/net/masterthought/cucumber/json/support/Status.java#L17-L19